ci/Jenkinsfiles/release.groovy

```diff
@@ -273,6 +284,10 @@ pipeline {
           currentBuild.description = "Release ${RELEASE_VERSION} from build ${params.BUILD_VERSION}"
         }
       }
+      slackSend(channel: "${SLACK_CHANNEL}", color: "good", message: "Successfully promoted ${BRANCH_NAME} from ${CURRENT_VERSION} to ${RELEASE_VERSION}: ${BUILD_URL}")
+    }
+    failure {
+      slackSend(channel: "${SLACK_CHANNEL}", colo: "danger", message: "Failed to promote ${BRANCH_NAME} from ${CURRENT_VERSION} to ${RELEASE_VERSION}: ${BUILD_URL}")
```

Comment:
Condtioned by DRY_RUN?
And align message if updated.

Reply:
Done!